TVC_MVP/Alert/Alert.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Alert.swift
+//  TVC_MVP
+//
+//  Created by Shane McCully on 14/03/2019.
+//  Copyright © 2019 Shane McCully. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct TitleConstants {
+
+    static let dismiss = "Dismiss"
+    static let delete = "Delete"
+    static let tryAgain = "Try Again"
+
+}
+
+// NOTE: - AlertableView implementartion from DH
+
+protocol AlertView {
```

Comment:
We didn't mean for this to be brought over. The same can be achieved with a traditional way, and would be more beneficial for you to learn than bringing this handy implementation. You will be able to come to this on your own, only after you struggle implementing it the other way a couple of times.